docs/plans/2026-03-06-local-image-cache-integration-tests.md

```
@@ -0,0 +1,364 @@
+        }
+    }
+    .runWith(Sink.ignore)
+    .onComplete { _ =>
+      progressMap.put(contestId, mkProgress(done.get, running = false))
+    }
+}
+```
+
+**Step 4: Compile**
+
+```
```

markdownlint-cli2 (0.22.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)